usaspending_api/api_contracts/contracts/v2/recipient/state/awards.md

```diff
@@ -0,0 +1,31 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# State Award Breakdown [/api/v2/recipient/state/awards/]
+
+This endpoint is used to power USAspending.gov's state profile pages. This data can be used to visualize the government spending that occurs in a specific state or territory.
+
+## Get State Award Breakdown [GET /api/v2/recipient/state/awards/{fips}/{?year}]
+
+This endpoint returns the award amounts and totals, based on award type, of a specific state or territory, given its USAspending.gov `id`.
+
++ Parameters
+
+    + fips: 51 (required, string)
```

Comment:
```suggestion
    + fips: `51` (required, string)
```
Feel free to ask around and not take these if more people think it's better the other way. This is not a hill I'll die on.